tests/pot3d/testsuite/validation2/input/pot3d.dat

```diff
@@ -0,0 +1,30 @@
+ &topology
+  nr=63
+  nt=91
+  np=225
+ /
+ &inputvars
+  ifprec=1
+  option='ss'
+  r1=2.5
+  rfrac=0.0,1.0
+  drratio=2.5
+  nfrmesh=0
+  tfrac=0.00
+  dtratio=1.0
+  nftmesh=0
+  pfrac=0.00
+  dpratio=1.0
+  nfpmesh=0
+  phishift=0.
+  br0file='br_input_tiny_bin.bin'
+  phifile=''
+  brfile=''
+  btfile=''
+  bpfile=''
+  br_photo_file=''
+  ncghist=10
+  ncgmax=10000000 
+  epscg=1.e-8
```

Comment:
This is the only point of difference between `validation` testsuite (which has `epscg=1.e-12`) and `validation2` in this PR.